config.go

```diff
@@ -103,6 +105,7 @@ func NewConfig() *Config {
 		NetInterfaceExcludeRegex:        []string{},
 		NetInterfaceExcludeLoopback:     true,
 		SystemFields:                    []string{"uname", "os_kernel", "os_family", "os_arch", "cpu_model", "fqdn", "memory_total_B"},
+		HardwareInventory:               true,
```

Comment:
Here we set to `true` but the default in the config is `false`?

Reply:
typo, done